test/development/acceptance/hydration-error.test.ts

```diff
@@ -73,52 +73,27 @@ describe('Error overlay for hydration errors in Pages router', () => {
     )
     const { session, browser } = sandbox
 
-    await session.assertHasRedbox()
-    expect(await getRedboxTotalErrorCount(browser)).toBe(isReact18 ? 2 : 1)
-
-    if (isReact18) {
-      expect(await session.getRedboxDescription()).toMatchInlineSnapshot(
-        `"Text content did not match. Server: "server" Client: "client""`
-      )
-    } else {
-      expect(await session.getRedboxDescription()).toMatchInlineSnapshot(
-        `"Hydration failed because the server rendered HTML didn't match the client. As a result this tree will be regenerated on the client. This can happen if a SSR-ed Client Component used:"`
-      )
-    }
-
-    if (isReact18) {
-      expect(await session.getRedboxDescriptionWarning()).toMatchInlineSnapshot(
-        `null`
-      )
-    } else {
-      expect(await session.getRedboxDescriptionWarning())
-        .toMatchInlineSnapshot(`
-          "- A server/client branch \`if (typeof window !== 'undefined')\`.
-          - Variable input such as \`Date.now()\` or \`Math.random()\` which changes each time it's called.
-          - Date formatting in a user's locale which doesn't match the server.
-          - External changing data without sending a snapshot of it along with the HTML.
-          - Invalid HTML tag nesting.
-
-          It can also happen if the client has a browser extension installed which messes with the HTML before React loaded."
-        `)
-    }
-    expect(await session.getRedboxErrorLink()).toMatchInlineSnapshot(
-      `"See more info here: https://nextjs.org/docs/messages/react-hydration-error"`
-    )
-
-    const pseudoHtml = await session.getRedboxComponentStack()
-
+    // Pages Router uses React version without Owner Stacks hence the empty `stack`
     if (isReact18) {
-      expect(pseudoHtml).toMatchInlineSnapshot(`
-       "<Mismatch>
+      await expect(browser).toDisplayRedbox(`
+       {
+         "componentStack": "<Mismatch>
```

Comment:
do you feel we should rename this property to sth else like `hydrationDiff`  since component stack sounds quite general. not blocking

Reply:
"diff" isn't the right word either since we also use this for stacks with invalid dom nesting. There's no diffing happening in those displays.